apiserver/backrestservice/backrestimpl.go

```diff
@@ -593,3 +623,18 @@ func createRestoreWorkflowTask(clusterName, ns string) (string, error) {
 	}
 	return spec.Parameters[crv1.PgtaskWorkflowID], err
 }
+
+// clusterNamesToPGClusterList takes a list of cluster names as specified by a slice of
+// strings containing cluster names and then returns a PgclusterList containing Pgcluster's
+// corresponding to those names.
+func clusterNamesToPGClusterList(clusterNames []string, namespace string) (crv1.PgclusterList,
```

Comment:
For newer stuff, I've been trying to reverse the order of namespace/cluster, i.e.

`namespace string, clustersNames []string`

as that seems to follow the hierarchical flow that we see when querying things

Reply:
Param order switched.  Namespace is now first.